From 742beae7efef23fb1df92afde0de5e1495c0936a Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 23 Dec 2022 13:18:08 +0200
Subject: [PATCH] [SW-113135] gaudi3: update type hardcoding in bootinfo set fn

Currently, bootinfo set function used in gaudi3 has type set to
PPBOOT_VERSION. Since boot info set function will be used for other
type, remove hardcoding of type and use type passed in function.

Change-Id: I0ac4417b96710a84193b1827f89b8c323d6093e3
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index b1da815..7ff38b1 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -162,8 +162,8 @@
 		mem_base += desc->size;
 	}
 
-	if (boot_info_set_no_merge(HL_BOOT_INFO_PPBOOT_VERSION, data, size)) {
-		preboot_puts("boot_info_set: failed to set HL_BOOT_INFO_PPBOOT_VERSION\n");
+	if (boot_info_set_no_merge(type, data, size)) {
+		preboot_puts_hex("boot_info_set: failed to set boot info type:", type, false);
 		return 1;
 	}
 
